models/gc_wsi_bgseg/meta.json

```diff
@@ -0,0 +1,156 @@
+    "use": {
+      "title": "Intended use",
+      "text": "This algorithm is a segmentation method of the background on whole-slide histopathology images. The algorithm will segment WSI with various stains and resolutions. It takes pixel spacing as input parameter (default 2.0μm) and will create a segmentation of that resolution. The input WSI needs to contain at least one layer of that resolution, with a 25% tolerance. The algorithm can also be found on Grand Challenge [1] (access with verified account).",
+      "references": [
+        {
+          "label": "Algorithm on Grand Challenge (access with verified account)",
+          "uri": "https://grand-challenge.org/algorithms/tissue-background-segmenation"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "Evaluation was determined with DICE score, sensitivity, and false-positive count, and compared with three other non-deeplearning methods: Otsu's, FESI, and Thresholding. See the paper for additional details [1].",
+      "references": [
+         {
+          "label": "Resolution-agnostic tissue segmentation in whole-slide histopathology images with convolutional neural networks",
+          "uri": "https://peerj.com/articles/8242/"
+        }
+      ],
+      "tables": [
+        {
+          "label": "Dice scores at 2.0μm pixel spacing.",
+          "entries": {
+            "Thresholding": "0.8627 ±0.1361",
+            "Otsu's": "0.7373 ±0.1596",
+            "FESI": "0.8284 ±0.3288",
+            "This model": "0.9822 ±0.0195"
+          }
+        },
+        {
+          "label": "Sensitivity scores at 2.0μm pixel spacing.",
+          "entries": {
+            "Thresholding": "0.5763 ±0.3631",
+            "Otsu's": "0.2890 ±0.3665",
+            "FESI": "0.6495 ±0.3690",
+            "This model": "0.8953 ±0.1302"
+          }
+        },
+        {
+          "label": "False-positive count at 2.0μm pixel spacing.",
+          "entries": {
+            "Thresholding": "34.50 ±76.49",
+            "Otsu's": "12.63 ±12.09",
+            "FESI": "1.37 ±2.63",
+            "This model": "5.37 ±8.25"
+          }
+        }
+      ]
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The evaluation was performed on two datasets. The first dataset consisted of 30 WSI with breast, lymph node, kidney, lung, rectum and tongue tissue, stained with hematoxylin and eosin (H&E), Sirius Red, Periodic Acid-Schiff (PAS), cytokeratin AE1/AE3 (AE1AE3), Ki-67, and a cocktail of cytokeratin 8 and cytokeratin 18 (CK8-18). The second dataset consisted of 8 WSI with mostly different tissue types and all different staining methods, to evaluate generalization of the model. It contained tissue types lung, cornea, aorta, brain, skin, uterus, and kidney. The tissues were stained with Grocott, Alcian Blue, Von Kossa, Perls, and Chromotrope Aniline Blue (CAB) stains. See the paper for additional details [1].",
```

Comment:
Referencing the paper is fine, it'll be nice to mention where to look in the paper (e.g., appendix X, page Y, section A.B, etc.)